static/docs/commands-reference/repro.md

```diff
@@ -1,7 +1,7 @@
 # repro
 
-Reproduce a stage that is specified by a `.dvc` file and all stages the file
-lists as dependencies (recursively).
+Rerun all commands recorded in the pipeline in the same order, 
+or rerun specific commands associated with stages named on the command line.
```

Comment:
You are suggesting a circular definition, that the "repro" command is defined as "reproducing".  To avoid the circular definition I expanded the meaning, because "reproduce" more or less means to run all the stages in the pipeline.

Reply:
I agree that we can change reproduce to its actual meaning - rerun the commands. It's still this definition (and the one below) lacks some important details. It does not run _all_ stages (unless you force it). By default it analyzes dependencies recursively and runs only necessary stages. The same when stage name is specified - by default it runs recursively.

Reply:
What I just wrote is "Rerun selected commands recorded in the pipeline in the same order, 
depending on command line options and the current workspace status."

Reply:
Looks good 👍  May be even better w/o selected - just simply it to `Run commands ....`, what do you think?

Reply:
Have updated the file to match